com-api/com-api-concept/src/lib.rs

```diff
@@ -249,10 +249,24 @@ where
     ///
     /// The caller has to make sure to initialize the data in the buffer before calling this method.
     unsafe fn assume_init(self) -> Self::SampleMut;
+
     /// Write a value into the buffer and render it initialized.
     ///
     /// This corresponds to `MaybeUninit::write`.
     fn write(self, value: T) -> Self::SampleMut;
+
+    /// Writes in place (skipping intermediate moves) the default value directly into the buffer and renders it initialized.
+    /// This requires that T implements `PlacementDefault`.
+    fn write_default(mut self) -> Self::SampleMut
+    where
+        Self: Sized,
+        T: PlacementDefault + Send + Reloc + Debug,
```

Comment:
` + Send + Reloc + Debug` is redundant with the constraints on `T` above

```suggestion
        T: PlacementDefault,
```